bdk_chain/tests/test_keychain_txout_index.rs

```diff
@@ -24,6 +24,7 @@ fn init_txout_index() -> KeychainTxOutIndex<TestKeychain> {
 }
 
 #[test]
+#[cfg(feature = "miniscript")]
```

Comment:
this whole file should have `#![cfg(feature = "miniscript")]` I think I also had to do that in one of my other PRs.